judo-ui-react/src/main/java/hu/blackbelt/judo/ui/generator/react/UiGeneralHelper.java

```diff
@@ -50,6 +50,10 @@ public static String plainName(String input) {
         return input.replaceAll("[^\\.A-Za-z0-9_]", "_").toLowerCase();
     }
 
+    public static String getApplicationConfigRestPathName(ClassType classType) {
+        return classType.getFQName().replaceAll("[#\\.]", "/").replace("[^0-9a-zA-Z_~/]", "_");
+    }
+
```

Comment:
Not used

Reply:
no way

Reply:
[715d4b9](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/235/commits/715d4b9dfdf0d0937041c90042ddf2ed26e117fc)